### PR TITLE
create passwordService test

### PR DESCRIPTION
**Pull request by bendesbeus**

### CODE REVIEW

File: sk0-back/src/users/services/password.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    expect(service).toBeDefined();
+  });
+
+  describe('hash', () => {
+    it('should hash a password', async () => {
+      const password = 'examplePassword';
+      const hashedPassword = await service.hash(password);
+      expect(hashedPassword).toBeDefined();
+      expect(hashedPassword.length).toBeGreaterThan(0);
+    });
```
Comment (markkovari) [inline sk0-back/src/users/services/password.service.spec.ts]: This is a good test, if you want you can extend it with another `expect` to see that the original password does not equal to the `hashedPassword`. We should not use a hashing which has the same input and output.

```suggestion
      expect(hashedPassword.length).toBeGreaterThan(0);
      expect(hashedPassword).not.ToEqual(password);
    });
```

**Review by markkovari — COMMENTED** [review]
Nice PR 💪 , if you allow I would add a not equals to the originl-created pair too.

File: sk0-back/src/users/services/password.service.spec.ts (hunk 1 above)
Comment (bendesbeus) [reply sk0-back/src/users/services/password.service.spec.ts]: good point, i'm on it

**Review by Jankita58 — APPROVED** [review]

**Review by markkovari — APPROVED** [review]

**Review by rolcho — APPROVED** [review]
nice!

**Pull request merged**